### PR TITLE
Caching layer in Elastic Search - add new index

### PR DESCRIPTION
**Pull request by bartlomiejmarszal**
Due to required new queries I introduced additional constants and block reformat. 
- Additional indexes added

[CON-496](https://oat-sa.atlassian.net/browse/CON-496)

### CODE REVIEW

File: src/IndexerInterface.php [hunk 1]
```diff
@@ -40,6 +40,12 @@ interface IndexerInterface
     public const UNCLASSIFIEDS_DOCUMENTS_INDEX = 'unclassifieds';
     public const ASSETS_INDEX = 'assets';
 
+    //Properties
```
Comment (boajer) [inline src/IndexerInterface.php:43]: Do we need this line?

File: src/IndexerInterface.php [hunk 2]
```diff
@@ -52,6 +58,7 @@ interface IndexerInterface
         TaoOntology::CLASS_URI_SUBJECT => self::TEST_TAKERS_INDEX,
         TaoOntology::CLASS_URI_TEST => self::TESTS_INDEX,
         self::MEDIA_CLASS_URI => self::ASSETS_INDEX,
+        'property-list' => 'property-list'
```
Comment (boajer) [inline src/IndexerInterface.php:61]: Why we have this only hardcoded string at the array with a constant keys?

Comment (bartlomiejmarszal) [reply src/IndexerInterface.php:61]: This is very good question. **This string is representing new index** that I have to add here in order to be accepted by Indexer. This seems like huge restriction in library that I would like to highlight in order to fix it ASAP.

File: src/QueryBuilder.php [hunk 3]
```diff
@@ -40,6 +41,7 @@ class QueryBuilder extends ConfigurableService
         'tests' => IndexerInterface::TESTS_INDEX,
         'TestTaker' => IndexerInterface::TEST_TAKERS_INDEX,
         'taoMediaManager' => IndexerInterface::ASSETS_INDEX,
+        'property-list' => 'property-list'
```
Comment (gabrielfs7) [inline src/QueryBuilder.php:44]: We are following camel case, no?

**Review by gabrielfs7 — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by shpran — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Pull request merged**